apps/api/migrations/060_add_tamper_evident_audit_fields.down.sql

```
@@ -0,0 +1,13 @@
+DROP INDEX IF EXISTS idx_audit_logs_entry_hash;
```

SQLFluff (4.2.2)

[error] DROP statement should use CONCURRENTLY to avoid locking the table.

(PG01)